pom.xml

```diff
@@ -152,20 +162,20 @@
                         <!-- config is inherently not testable -->
                         <exclude>org/sagebionetworks/bridge/udd/config/*</exclude>
 
+                        <!-- no point in testing exceptions, since they are entirely boilerplate code -->
+                        <exclude>org/sagebionetworks/bridge/udd/exceptions/*</exclude>
```

Comment:
I just mean to verify your error messages are correct. Not that important but I've been finding validator messages that are incomplete or incorrect.